ReSharper.FSharp/src/FSharp.Psi.Features/src/Daemon/Stages/FSharpQuickDocSuppressor.fs

```diff
@@ -0,0 +1,33 @@
+namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Stages
+
+open JetBrains.DocumentModel.DataContext
+open JetBrains.ProjectModel
+open JetBrains.RdBackend.Common.Features.Daemon.Tooltips
+open JetBrains.RdBackend.Common.Features.Daemon.Tooltips.Request
+open JetBrains.ReSharper.Plugins.FSharp.Psi
+open JetBrains.ReSharper.Psi.DataContext
+open JetBrains.Rider.Backend.Features.QuickDoc.Impl
+open JetBrains.TextControl
+open JetBrains.TextControl.DataContext
+
+[<SolutionComponent>]
+type FSharpQuickDocSuppressor
```

Comment:
It seems to me that a constructor should be added here
```suggestion
type FSharpQuickDocSuppressor()
```

Reply:
@DedSec256 There's a constructor one line below.